packages/rbac/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@redhat-cloud-services/rbac-client",
   "version": "4.2.0",
-  "description": "",
+  "description": "TypeScript client for Red Hat Insights RBAC service",
```

Comment:
This change is only meant to trigger a release of previously failed `rbac-client` release.